src/shared/style/sv/utils.ts

```
@@ -0,0 +1,20 @@
+  styles.reduce((acc, style) => {
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
+    if ([null, false, 0, 0n].includes(style as any)) {
+      return acc;
+    }
+
+    if (Array.isArray(style)) {
+      return mergeStyleValues(acc, ...style);
+    }
+
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
+    return { ...acc, ...style } as any;
```

Biome (1.9.4)

[error] Avoid the use of spread (`...`) syntax on accumulators.

Spread syntax should be avoided on accumulators (like those in `.reduce`) because it causes a time complexity of `O(n^2)`.
Consider methods such as .splice or .push instead.

(lint/performance/noAccumulatingSpread)